data/bookkeeping/block.go

```diff
@@ -467,6 +489,61 @@ func ProcessUpgradeParams(prev BlockHeader) (uv UpgradeVote, us UpgradeState, er
 	return upgradeVote, upgradeState, err
 }
 
+type bonusPlan struct {
+	baseRound     basics.Round
```

Comment:
It might be good to document baseRound/baseAmount/decayInterval definitions/intent

Reply:
See if that works.  I also simplified the handling of decay, so it's easier to follow what happens when `bonusPlan` changes.